grpc_opa/authorizer.go

```diff
@@ -97,6 +104,9 @@ type Authorizer interface {
 	// OpaQuery executes query of the specified decisionDocument against OPA.
 	// If decisionDocument is "", then the query is executed against the default decision document configured in OPA.
 	OpaQuery(ctx context.Context, decisionDocument string, opaReq, opaResp interface{}) error
+
+	// String implements fmt.Stringer interface
+	String() string
```

Comment:
Same thing, interfaces are implicit in go. Adding a `String()` method is sufficient to implement fmt.Stringer